KOService.Presentation/KOService/src/app/shared/services/repair.service.ts

```diff
@@ -12,11 +14,14 @@ export class RepairService {
 
   private url = 'https://localhost:44340/api/repairs/';
 
-
   constructor(private httpClient: HttpClient) {
   }
 
-  getRepairs(managerId: string, statusQuery = ''): Observable<Repair[]> {
-    return this.httpClient.get<Repair[]>(this.url + managerId, {params: new HttpParams().set('status', statusQuery)});
+  getRepairs(statusQuery = ''): Observable<Repair[]> {
+    return this.httpClient.get<Repair[]>(this.url, { params: new HttpParams().set('status', statusQuery) });
+  }
+
+  addRepair(repair: RepairForCreation) {
```

Comment:
no to tuaj zwracasz Obsevable a w komponencie brakuje ci subscriba

Reply:
Dobra, popracowałem nad tym i prawie działa: brakuje tylko działającego pobierania id menadżera (w mojej nie działa, nie wiem jak w obecnej)

Reply:
to moze zmerguj sobie najnowsza wersje. I zobacz wtedy :P

Reply:
I tak musisz to zrobić bo masz sporo konflików